app/server/utils/ussd/token_processor.py

```diff
@@ -73,61 +74,53 @@ def get_default_limit(user: User, token: Token, transfer_account: TransferAccoun
         :param transfer_account:
         :return: lowest limit applicable for a given CreditTransfer
         """
```

Comment:
this is literally the same function, just wrapped in the ephemeral object context manager.

Reply:
so a credit transfer was being committed to the DB somehow?

Reply:
yep! undocumented SQLAlchemy behaviour

Reply:
👻